src/shell/lean.cpp

```diff
@@ -644,7 +644,7 @@ int main(int argc, char ** argv) {
         }
 
         if (c_output && ok) {
-            std::ofstream out(*c_output);
+            std::ofstream out(*c_output, std::ios_base::binary);
```

Comment:
what is this fixing?

Reply:
See commit message, we should produce the same `stage0/` on every platform. `EmitC` already uses `\n` explicitly, but the `ofstream` mangled it into `\r\n`.